MediaBrowser.Api/UserService.cs

```diff
@@ -379,6 +379,11 @@ public object Post(AuthenticateUser request)
                 throw new ResourceNotFoundException("User not found");
             }
 
+            if (request.Pw == "")
```

Comment:
I’m no genius, but I think you want a `string.IsNullOrEmpty(request.Pw)` here.

Reply:
Yes

Reply:
So
```suggestion
            if (string.IsNullOrEmpty(request.Pw))
```